packages/merge-styles/src/extractStyleParts.ts

```diff
@@ -1,48 +1,54 @@
 import { IStyle, IStyleBaseArray } from './IStyle';
 import { Stylesheet } from './Stylesheet';
 
+export interface IExtractStylePartsResult {
+  classes: string[];
+  objects: IStyleBaseArray;
+}
+
 /**
  * Separates the classes and style objects. Any classes that are pre-registered
  * args are auto expanded into objects.
  */
-export function extractStyleParts(
-  ...args: (IStyle | IStyle[] | false | null | undefined)[]
-): { classes: string[]; objects: IStyleBaseArray } {
-  const classes: string[] = [];
-  const objects: {}[] = [];
+export function extractStyleParts(...args: (IStyle | IStyle[] | false | null | undefined)[]): IExtractStylePartsResult {
```

Comment:
Should we also write tests for this?

Reply:
Would be ideal, yes. In this case this was just moving code and not writing anything new.